scripts/create-conda-recipe.py

```diff
@@ -0,0 +1,188 @@
+import re
+from collections import defaultdict
+from copy import deepcopy
+
+import requests
+import yaml
+from bs4 import BeautifulSoup
+
+
+def get_extra_requires(path):
```

Comment:
Can you add a test to test `get_extra_requires` and show what this function is doing?